.circleci/config.yml

```diff
@@ -191,6 +191,9 @@ jobs:
           name: Deploy to dev
           command: |
               ./blue-green-deploy.sh $CRT_USERNAME_DEV $CRT_PASSWORD_DEV dev
+      - run:
```

Comment:
I suspect this will fail since playwright and pytest are development dependencies and not installed here, we should be able to borrow the set-up steps from `build_and_test` above, bonus points if we can define it in a single place and re-use.